tt_eager/tt_dnn/op_library/eltwise_binary/eltwise_binary_op.cpp

```diff
@@ -198,9 +198,9 @@ std::vector<Tensor> EltwiseBinary::create_output_tensors(
         }
         auto mem_config = this->output_mem_config;
         mem_config.shard_spec = shard_spec;
-        return {create_sharded_device_tensor(this->compute_output_shapes(input_tensors).at(0), this->output_dtype, Layout::TILE, input_tensor_a.device(), mem_config)};
+        return {create_sharded_device_tensor(this->compute_output_shapes(input_tensors).at(0), tt::tt_metal::DataType::BFLOAT16, Layout::TILE, input_tensor_a.device(), mem_config)};
     }
-    return operation::generic_create_output_tensors(*this, input_tensors, this->output_dtype, Layout::TILE, this->output_mem_config);
```

Comment:
This change is not okay. Output_dtype is passed in by the user, or defaults to input dtype. If bfp8 does produce accurate results for certain ops, then user should specify bfloat16. We cant just hardcode the output to bfloat16.

Reply:
Removed the changes